android/build.gradle

```diff
@@ -1,8 +1,8 @@
-group 'vn.hunghd.flutterdownloader'
+group 'com.github.fluttercommunity.flutterdownloader'
```

Comment:
Let's use double quotes for all strings. It'll be easier to convert to Gradle Kotlin DSL in the future.